site/app/templates/forum/CreatePost.twig

```diff
@@ -120,7 +120,7 @@
                 {% if not (isThreadLocked != 1 or userAccessFullGrading == true) %}
                     {#pass#}
                 {% else %}
-                    <a class="dropdown-item key_to_click" tabindex = "1" data-testid="delete-post-button" onClick="deletePostToggle({{ post_buttons.delete.ud_toggle_status }}, {{ post.thread_id }}, {{ post.id }}, '{{ post.author_user_id }}', '{{ post_date }}', '{{ csrf_token }}' ); return false;" title="{{ post_buttons.delete.ud_button_title }}" aria-label="{{ post_buttons.delete.ud_button_title }}">Delete</a>
+                    <a class="dropdown-item key_to_click" tabindex = "1" data-testid="delete-post-button" onClick="deletePostToggle({{ post_buttons.delete.ud_toggle_status }}, {{ post.thread_id }}, {{ post.id }}, '{{ post.author_user_id }}', '{{ post_date }}', '{{ csrf_token }}' ); return false;" title="{{ post_buttons.delete.ud_button_title }}" aria-label="{{ post_buttons.delete.ud_button_title }}">{{ post_buttons.delete.ud_toggle_status == "true" ? "Delete" : "Undelete" }}</a>
```

Comment:
What do you think about changing the text to "Restore"?

Reply:
That's what I originally planned but it would be inconsistent everywhere else.